africastalking/Service.py

```diff
@@ -81,7 +82,7 @@ def _init_service(self):
 
     @staticmethod
     def __make_get_request(url, headers, data, params, callback=None):
-        res = requests.get(url=url, headers=headers, params=params, data=data)
+        res = requests.get(url=url, headers=headers, params=params, data=data, timeout=TIMEOUT)
```

Comment:
@felixcheruiyot would it not be better to have the `timeout` value come all the way from the exposed methods? e.g.
```
....
sms.send("Hello Message!", ["+2547xxxxxx"],  connectionOptions = { "timeoutSeconds": 10 })
```